coderd/templateversions_test.go

```diff
@@ -151,7 +157,7 @@ func TestPatchCancelTemplateVersion(t *testing.T) {
 			var err error
 			version, err = client.TemplateVersion(context.Background(), version.ID)
 			require.NoError(t, err)
-			return version.Job.Status == codersdk.ProvisionerJobCanceled
+			return version.Job.Status == codersdk.ProvisionerJobFailed // cancel times out
```

Comment:
Does anyone know why this might be timing out? I don't think this is related to the changes I made (but could be wrong). 
![Screen Shot 2022-07-20 at 10 32 35 AM](https://user-images.githubusercontent.com/19142439/180079203-ceef1f0a-1e74-4c4c-96f4-ae7b09399f5e.png)

Reply:
I checked on main (with some changes to remove that pesky `require.Eventually`) and it looks like this test is already subtly broken:

![image](https://user-images.githubusercontent.com/4949514/180197017-1fe3e229-eafc-424b-876f-d6c8ace2a1d5.png)

I've filed a separate issue https://github.com/coder/coder/issues/3083 for this; you can leave this as-is for now and I'll follow up on that.